web/forms.py

```
@@ -283,6 +284,23 @@
         return user
 
 
+class TailwindInput(forms.widgets.Input):
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)